README.md

```diff
@@ -193,7 +193,7 @@ ulid=# EXPLAIN ANALYZE INSERT INTO users (name) SELECT 'Client 2' FROM generate_
 Use the extension in the database:
 
 ```sql
-CREATE EXTENSION ulid;
+CREATE EXTENSION pgx_ulid;
```

Comment:
Isn't this still the same as "ulid"?

Reply:
Not in my case, at least.

```
postgres=# create extension ulid;
ERROR:  extension "ulid" is not available
DETAIL:  Could not open extension control file "/share/postgresql/extension/ulid.control": No such file or directory.
HINT:  The extension must first be installed on the system where PostgreSQL is running.
postgres=# create extension pgx_ulid;
CREATE EXTENSION
```

I am building pgx_ulid with the default Nix builder `buildPgrxExtension` here is [the relevant builder's installation phase](https://github.com/NixOS/nixpkgs/blob/9bb668a7036c2ec6b2ca4234c6940c9b2df32049/pkgs/development/tools/rust/cargo-pgrx/buildPgrxExtension.nix#L164). And I don't see anything special/problematic about it, but it is the first time I am building something with pgrx, so I might be wrong!

Reply:
Ah, that is an issue with nix packaging for this. As you can see in our release process, we rename the files (https://github.com/pksunkara/pgx_ulid/blob/master/.github/workflows/release.yml#L80).

Reply:
Oh, oops, sorry, my silly mistake.

Neither mine derivation nor the one in the stalled PR to nixpkgs handles it correctly, since it was opened before the rename. 
Might try helping get it merged later. Thank you.